imgui-SFML.cpp

```diff
@@ -581,7 +584,8 @@ void Image(const sf::Texture& texture, const sf::Vector2f& size,
 
     ImTextureID textureID =
         convertGLTextureHandleToImTextureID(texture.getNativeHandle());
-    ImGui::Image(textureID, size, uv0, uv1, tintColor, borderColor);
+    ImGui::Image(textureID, ImVec2(size.x, size.y), uv0, uv1, ImColor(tintColor.toInteger()),
```

Comment:
Here and everywhere else you don't need to call `toInteger` on sf::Color

Reply:
toInteger on sf::Color is neccesarry otherwise it won't compile in visual studio.

Reply:
Wait, actually you need to use  toImColor wrapper here and everywhere else now